vanilla-framework/vanilla-framework
# 1510: Remove color and background from p-list-step__content

## Done

- Removed color and background from p-list-step__content so it inherits from parent element

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/lists/lists-stepped-detailed/
- Open dev tools and add `<div class="p-strip--dark"><div class="row">` under the `<body>` tag and `</div></div>` above the `</body>` tag
- Check that the list content has the same background and font color as the headings

## Details

Fixes #1510 

## Screenshots
### Before
![screenshot](https://user-images.githubusercontent.com/25733845/34714634-f71eca38-f521-11e7-8e00-3b667dac17b5.png)

### After
![screenshot2](https://user-images.githubusercontent.com/25733845/34714647-ff4fb85c-f521-11e7-9c7f-5f9a0747c54f.png)
